### PR TITLE
Enable static `use` and `use_package`

### PR DESCRIPTION
**Pull request by eob**
Enables the following four methods:

Two static methods, which creates/loads Space `instance_handle` and then creates/loads the package/plugin instance `instance_handle` within it:

```
Steamship.use(package_handle, instance_handle)
Steamship.use_plugin(plugin_handle, instance_handle)
```

Two instance methods, which use the CURRENT active space and create/load the package/plugin instance `instance_handle` within it:

```
client.use(package_handle, instance_handle)
client.use_plugin(plugin_handle, instance_handle)
```

This is a huge usability improvement for quick, simple, idempotent package loading. In a Jupyter notebook or anywhere in your code, you can simply say:

```
instance = Steamship.use('package-name', 'my-instance-name')
```

And you always get (1) the same instance, (2) running in its own workspace.

### CODE REVIEW

File: src/steamship/client/client.py [hunk 1]
```diff
@@ -108,20 +124,52 @@ def embed_and_search(
...
+        For example, one may write the following to always get back the same package instance, no matter how many
+        times you run it, scoped into its own workspace:
+
+        ```python
+        plugin = Steamship.use('package-handle', 'instance-handle')
+        ```
+        """
+        kwargs["workspace"] = instance_handle
+        client = Steamship(**kwargs)
+        return client.use(
```
Comment (EniasCailliau) [inline src/steamship/client/client.py:149]: I would personally use `_instance_use` here as this eases code navigation. Took me a while to understand/remember that use is actually pointing to `_instance_use`.

**Review by EniasCailliau — APPROVED** [review]
1 small suggestion. Thanks for taking the time to write clear documentation!

File: src/steamship/client/client.py [hunk 2]
```diff
@@ -108,20 +124,52 @@ def embed_and_search(
             space=space,
         )
 
+    @staticmethod
     def use(
+        package_handle: str,
+        instance_handle: str = None,
```
[the remark this answers is not in the document]
Comment (eob) [reply src/steamship/client/client.py:130]: Done

File: src/steamship/client/client.py [hunk 3]
```diff
@@ -130,24 +178,56 @@ def use(
             raise instance.error
         if not instance.data:
             raise SteamshipError(
-                f"Unable to create an instance of App {app_handle} with handle {handle}."
+                f"Unable to create an instance of App {package_handle} with handle {instance_handle}."
```
[the remark this answers is not in the document]
Comment (eob) [reply src/steamship/client/client.py:181]: Done

File: src/steamship/client/client.py (hunk 1 above)
Comment (eob) [reply src/steamship/client/client.py:149]: Done

**Pull request merged**